packages/contractkit/src/wrappers/Validators.ts

```diff
@@ -48,6 +48,19 @@ export class ValidatorsWrapper extends BaseWrapper<Validators> {
     toNumber
   )
 
+  async validatorSetAddresses(): Promise<string[]> {
+    const numberValidators = await this.numberValidatorsInCurrentSet()
+
+    const validators = []
+
+    for (let i = 0; i < numberValidators; i++) {
```

Comment:
If you want to make @nambrot happy... you can do

```ts
const validators = await Promise.all(
  Array.from(Array(numberValidators).keys()).map(i => this.validatorAddressFromCurrentSet(i))
)
```

and run calls in parallel

Reply:
The "range" array comes from https://stackoverflow.com/questions/3746725/how-to-create-an-array-containing-1-n?page=1&tab=votes#tab-top

and it's a good fn for utils :P

Reply:
So the whole `Array.from...` thing for creating a range in js is kind of unfortunate, think a for loop is the lesser evil. But definitely can `Promise.all` this stuff!